akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -538,7 +578,9 @@ private[persistence] trait Eventsourced extends Snapshotter with Stash with Stas
         // while message is in flight, in that case we ignore the call to the handler
         if (id == instanceId) {
           try {
+            insidePersistCallback = true
             pendingInvocations.peek().handler(l)
+            insidePersistCallback = false
```

Comment:
what happens if the handler throws, and resume supervision is used?
should this be done in a finally?